.azure-pipelines/auto.yml

```diff
@@ -126,27 +126,63 @@ jobs:
 
       asmjs:
         IMAGE: asmjs
-      i686-gnu:
+
+      i686-gnu-1:
+        IMAGE: i686-gnu
+        SCRIPT: make ci-subset-1
+      i686-gnu-2:
         IMAGE: i686-gnu
-      i686-gnu-nopt:
+        SCRIPT: make ci-subset-2
+
+      i686-gnu-nopt-1:
+        IMAGE: i686-gnu-nopt
+        SCRIPT: make ci-subset-1
+      i686-gnu-nopt-2:
         IMAGE: i686-gnu-nopt
+        SCRIPT: make ci-subset-2
+
       test-various:
         IMAGE: test-various
-      x86_64-gnu:
+
+      x86_64-gnu-1:
         IMAGE: x86_64-gnu
+        SCRIPT: make ci-subset-1
+      x86_64-gnu-2:
+        IMAGE: x86_64-gnu
+        SCRIPT: make ci-subset-2
+
       x86_64-gnu-full-bootstrap:
         IMAGE: x86_64-gnu-full-bootstrap
+
       x86_64-gnu-aux:
         IMAGE: x86_64-gnu-aux
+        SCRIPT: make check-aux EXCLUDE_CARGO=1
+
+      x86_64-gnu-cargo:
```

Comment:
Did we forget to include this from before?

Reply:
No, on Travis `x86_64-gnu-cargo` is included inside `x86_64-gnu-aux`. Instead on AppVeyor, to speed things up, cargo is tested inside its own job. I replicated the same thing we do on AppVeyor in Azure Linux.